src/utils/ReentrancyGuard.sol

```diff
@@ -8,7 +8,9 @@ abstract contract ReentrancyGuard {
     uint256 private locked = 1;
 
     modifier nonReentrant() virtual {
-        require(locked == 1, "REENTRANCY");
+        //`require(locked == 1)` works fine in normal conditions. But when deployed via proxy, the storage slot should be manually set to 1, by default it would be 0.
```

Comment:
OCD Nit: Add a space after the first `//`. 

Break up into shorter lines. Keep things below 100 characters or 80 characters. 

This is so that we can read the code easily on a phone.

Sent from my iPhone while sitting on the toilet bowl.